setup/context/commands.go

```diff
@@ -63,8 +87,21 @@ func Apply(c *Context, force bool) error {
 		}
 	}
 
-	// write empty dockercfg file if it doesn't exist
-	if err := checkDockercfg(c, force); err != nil {
+	var dockercfg string
+	dockercfg_flag, ok := c.Flags["dockercfg"]
```

Comment:
`dockercfg` should not be a context flag, it should be passed in as a param to `Apply` just like `force` flag is (will go over more in main)